browser/gui/app.cpp

```diff
@@ -123,6 +123,12 @@ App::App(std::string browser_title, std::string start_page_hint, bool load_start
         spdlog::critical("imgui-sfml initialization failed");
         std::abort();
     }
+    // ImGui tries to write to the current working directory. This can be problematic for a number of reasons,
+    // so it can be prevented via an env var.
+    if (std::getenv("HASTUR_FORBID_DISKIO")) {
```

Comment:
`HST_` instead of `HASTUR_` to match the env var (`HST_SCALE`) we already support, and `DISK_IO` since that's not one word.